revolv/administrator/views.py

```diff
@@ -28,6 +31,18 @@ def get_context_data(self, **kwargs):
         context['subscribed_user_emails'] = user_emails
         return context
 
+class AdministratorAddAdjustmentView(CreateView):
+    
+    model = AdminAdjustment
+    template_name = 'administrator/adjustment.html'
+
+    def get_success_url(self):
+        return reverse('administrator:accounting')
+
+    def get_form_kwargs(self):
+        kwargs = super(AdministratorAddAdjustmentView, self).get_form_kwargs()
+        self.form = AdjustmentForm()
```

Comment:
I'm reading the source of CreateView and I can't seem to find why you would need to set self.form here - why?